### PR TITLE
Feature/1928 navbar icon animtaion

### LINKED ISSUE #1928: Desktop: Fix nav bar items behaviour and click area
### Issue 
On Desktop, the language, feedback and profile icons in the nav bar have irregular and too small click areas which makes it hard to interact with. Also these icons do not behave as the rest when hovered on.

Correct:
image

Incorrect:
image
image

### To Fix
Feedback, language and profile icons should have their click areas defined as the rest of the items like messaging and notifications do, and they should behave the same on hover.

### PR DESCRIPTION
**Pull request by chrislemus**
### What does this pull request aim to achieve? 💯

_Please be concise and link any related issue(s): resolve issue #1928

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)
#1928

### CODE REVIEW

File: client/src/components/Feed/MessagesModal/MessageModal.js [hunk 1]
```diff
@@ -68,7 +68,7 @@ const MessageModal = ({
       showModal();
     }
     sessionStorage.removeItem("msgModal");
-  }, [])
+  }, [authorId, isAuthenticated]);
```
Comment (Zaydme) [inline client/src/components/Feed/MessagesModal/MessageModal.js]: This should not be added here 😅
I assume it was added by the linter.
Please clear the dependency array and add the eslint ignore rule.

**Comment by Zaydme** [comment]
And a little bit of misalignment
Screenshot_20201128-230900_Chrome
Other than that, it LGTM! Thanks @chrislemus

File: client/src/components/Feed/MessagesModal/MessageModal.js [hunk 2]
```diff
@@ -68,7 +68,8 @@ const MessageModal = ({
       showModal();
     }
     sessionStorage.removeItem("msgModal");
-  }, [])
+    // eslint-disable-next-line no-use-before-define
```
Comment (Zaydme) [inline client/src/components/Feed/MessagesModal/MessageModal.js]: Sorry for not specifying, my bad.
But this is the rule we need there `// eslint-disable-next-line react-hooks/exhaustive-deps`
(Also you will need to empty the array in the next line)

**Comment by rabbi08** [comment]
@chrislemus 
While verifying the fix, would like to highlight below observations:

Screen Shot 2020-11-28 at 11 54 57 PM

**Review by Zaydme — APPROVED** [review]
LGTM! Thanks @chrislemus

**Review by idjevm — APPROVED** [review]
LGTM!

**Pull request merged**